thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -1188,7 +1188,7 @@ class CInMemJoinBase : public CSlaveActivity, public CAllOrLookupHelper<HELPER>,
             const void *rightRow = numRows ? filteredRhs.item(0) : defaultRight.get();
             if (isGroupOp())
             {
-                size32_t sz = HELPERBASE::joinTransform(rowBuilder, leftRow, rightRow, numRows, filteredRhs.getArray());
+                size32_t sz = HELPERBASE::joinTransform(rowBuilder, leftRow, rightRow, numRows, filteredRhs.getArray(), JTFmatchedleft|JTFmatchedright);
```

Comment:
The if at the top of this block checks "if (filteredRhs.ordinality()"
I think the line:
            const void *rightRow = numRows ? filteredRhs.item(0) : defaultRight.get();
could also be simplified.